<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>12</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="9" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="gb_png_setting">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Setting for png</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cb_forcePngAlpha">
        <property name="text">
         <string>Clear image for Twitter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_aggressivePngCompress">
        <property name="text">
         <string>Compress with PngQuant(need pngquant.exe)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l_filename_6">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="sb_quality">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QCheckBox" name="cb_overwriteConfirm">
     <property name="text">
      <string>Confirm when overwrite existing file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="le_filename"/>
   </item>
   <item row="15" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_finish">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="QCheckBox" name="cb_stayOnTop">
     <property name="text">
      <string>Always stay on top</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="l_filename_5">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Language
(needs restart)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="l_filename_2">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Delayed time(sec)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_filename">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>File name format</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="l_path">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="b_folder">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QComboBox" name="cb_language">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="cb_filetype">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bmp</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="l_folder">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Folder</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="lb_filenameSample">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QCheckBox" name="cb_openDialog">
     <property name="text">
      <string>Open folder after saving screen shot</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="l_filename_4">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSpinBox" name="sb_delaySecs">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QCheckBox" name="cb_drawArrow">
     <property name="text">
      <string>Draw arrow image on screen shot</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="l_filename_3">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;</string>
     </property>
     <property name="text">
      <string>File format</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
